lxd/provider.go

```diff
@@ -217,7 +226,7 @@ var descriptions map[string]string
 func init() {
 	descriptions = map[string]string{
 		"lxd_accept_remote_certificate":    "Accept the server certificate",
-		"lxd_config_dir":                   "The directory to look for existing LXD configuration. default = $HOME/.config/lxc",
+		"lxd_config_dir":                   "The directory to look for existing LXD configuration. default = $HOME/snap/lxd/common/config",
```

Comment:
Dunno if that make sense but we could use a similar notation as for `$PATH` in `shell` so `default = $HOME/snap/lxd/common/config:$HOME/.config/lxc"`